reconcile/aws_iam_keys.py

```diff
@@ -43,8 +43,13 @@ def cleanup(working_dirs):
 
 @defer
 def run(dry_run, thread_pool_size=10,
-        disable_service_account_keys=False, defer=None):
+        disable_service_account_keys=False, account_name=None, defer=None):
     accounts = queries.get_aws_accounts()
+    if account_name:
```

Comment:
Can we please add a test for this kind of filter? Other than this, it looks good.

Reply:
can we do it without writing a test for the entire integration?

Reply:
Mmm... you could move this logic to its own method, say `filter_accounts` and test that. Or it's not worth it and we can write a full test suite in a different PR. Up to you.

Reply:
extracted to function and added tests!